frontend/src/metabase/lib/i18n.js

```diff
@@ -46,12 +46,6 @@ function updateMomentStartOfWeek() {
   if (startOfWeekDayNumber === -1) {
     return;
   }
-  console.log(
-    "Setting moment.js start of week for Locale",
-    moment.locale(),
-    "to",
-    startOfWeekDayName,
-  );
```

Comment:
I removed this because the `withInstanceLocalization` needs to rapidly switch the locale and back, and this line causes a lot of console spam when this happens.